pkgs/by-name/re/restman/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, testers
+, restman
+}:
+
+buildGoModule rec {
+  pname = "restman";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    repo = "restman";
+    owner = "jackMort";
+    rev = "v${version}";
+    hash = "sha256-7Ebc/t4Z8EttdxtYrZrgXUIwPOYnSB/flK+sKH6bE28=";
+  };
+
+  vendorHash = "sha256-+a4+OYsLYozH2t6m3fC9WxZ30fxT9EPgpG11zQTSNmc=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+  ];
+
+  doInstallCheck = true;
+
+  passthru.tests = {
+    version = testers.testVersion {
+      package = restman;
+      version = "restman version ${version}";
+      command = "restman --version";
+    };
+  };
+
+  meta = with lib; {
+    description = "A sleek CLI for streamlined RESTful API testing and management. ";
+    homepage = "https://github.com/blampe/goat";
+    license = licenses.gpl3;
```

Comment:
```suggestion
    license = licenses.gpl3Only;
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/275812/commits/830e8978142303839e8c9155978685842b7ab8dc